dcs/Makefile

```diff
@@ -0,0 +1,52 @@
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+# This Makefile is just a thin shell to Maven, which is used to do the real build
+
+include ../core/macros.gmk #top level
+BLD_TRAFODION_DCS_TARNAME           =dcs-$(TRAFODION_VER).tar.gz
+
+VFILE			    =trafodion-dcs.jar.versions
+GENVERS			    =./genvers
+
+.NOTPARALLEL: all
+
+all: build_all
+
+build_all:	 
+	$(MAKE) build_chk
+	echo "$(MAVEN) site package -DskipTests"
+	echo "### For full Maven output, see file build_dcs.log"
+	set -o pipefail && $(MAVEN) site package -DskipTests | tee build_dcs.log | grep --line-buffered -E -e '^\[[^WId]' -e '^\[INFO\] B[Uu][Ii][Ll][Dd]' -e 'to compile'
+	mkdir -p ../${DISTRIBUTION_DIR}
+	cp -pf target/$(BLD_TRAFODION_DCS_TARNAME) ../${DISTRIBUTION_DIR}
```

Comment:
In other places, the tar file gets moved, instead of copied. Maybe we can save some space by moving?